params/config.go

```diff
@@ -381,6 +391,8 @@ func (c *ChainConfig) CheckConfigForkOrder() error {
 		{name: "ethTxTypeBlock", block: c.EthTxTypeCompatibleBlock},
 		{name: "magmaBlock", block: c.MagmaCompatibleBlock},
 		{name: "koreBlock", block: c.KoreCompatibleBlock},
+		{name: "kip103Block", block: c.Kip103CompatibleBlock},
```

Comment:
@ian0371 I didn't know it was intended.. Thanks for letting me know about it.

Reply:
It's what I thought initially so I need double check...

Reply:
If it is an optional fork, it looks natural to me not to check this fork ordering :)

Reply:
@yoomee1313 We've intentionally missed kip103Block in the fork ordering check because kip103 is designed as an optional hardfork and there are no dependence with other forks.

Reply:
We may leave a comment in our source code for the future developers who doesn't know about the context